xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
API is broken for Krypton anyways. Also, it doesnt really break with this change since setInfo("credits") will just go to /dev/null. Imo we dont need to go through the depracation-process. @ronie what do you think?